Classes/PassiveTreeView.lua

```diff
@@ -93,10 +93,12 @@ function PassiveTreeViewClass:Save(xml)
 	}
 end
 
-function PassiveTreeViewClass:Draw(build, viewPort, inputEvents)
+function PassiveTreeViewClass:Draw(build, compareSpec, viewPort, inputEvents)
```

Comment:
After looking at the code around other `Draw` functions, none of the others have data this specific in them.  They only pass in the viewPort most of the time, along with `inputEvents`.  Most of the time we pass the value needed to the `PassiveTreeView` from the calling class.  E.g. in `TreeTab.lua` you could do `self.viewer.compareSpec = compareSpec` instead of passing it to this function.  See `showHeatMap` for an example, too.

If we could refactor this slightly to follow the existing convention that'd be great.  Then you don't have to insert `nil` as part of the method call in the other tabs (Sorry, I looked at the behavior before the code, otherwise I would've suggested this before you made those changes!)